### ISSUE TITLE
Injection doesn't seem work when foldername starts with an uppercase character

### ISSUE THREAD

**Issue by nilsnilsnils** [johnno1962/InjectionIII#95]
Using App Store Version, XCode 10.1, macOS mojave

When I am saving the swift file the following error is shown in the console:

💉 *** Compiling /Users/XXXXX/Documents/git/project/MainCell.swift ***
💉 *** Re-compilation failed (/Users/XXXX/Library/Containers/com.johnholdsworth.InjectionIII/Data/command.sh)
<unknown>:0: error: error opening input file '/Users/XXXX/Documents/git/editor/SelectionViewController.swift' (No such file or directory)
 ***

The path to the  `SelectionViewController` is  '/Users/XXXX/Documents/git/Editor/SelectionViewController.swift'

The foldername starts with an **uppercase** character -> `E` (Editor)

**Comment by johnno1962** [comment]
Hi, looks like you found the answer. The recompilation in injection is case sensitive.

**Comment by johnno1962** [comment]
It is not possible to fix this. You need to make sure your project/workspace follows the correct capitalisation when you add the file.

**Comment by nilsnilsnils** [comment]
You mean that I need to rename all my folders in my project?

e.g. rename
'/Users/XXXX/Documents/git/Editor/SelectionViewController.swift'
to
'/Users/XXXX/Documents/git/editor/SelectionViewController.swift'

====
The files above are already in the project and I only edited them.

**Comment by johnno1962** [comment]
Renaming the folder is probably going to cause you confusion with git. The easier thing to do is carefully edit the 'project.pbxproj' project file in your project/workspace.

**Comment by nilsnilsnils** [comment]
Okay I found it.

I saw the "wrong" path in the XCode Inspector on the right side.
The path was not the same one as in the file system.

I re add all files which causes this problem and now a new problem occur:

💉 *** Link failed, check /Users/XXXX/Library/Containers/com.johnholdsworth.InjectionIII/Data/command.sh
ld: warning: -undefined dynamic_lookup is deprecated on iOS Simulator
ld: illegal text reloc in '_$S7Base16HomePageFlapCellC7setText4textySSSg_tF' to '_$S7Base12TextFlapViewC10_textLabelSo7UILabelCvpWvd' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
 ***

Any ideas ?

Thanks for your help.
